src/gui/appclient.cpp

```diff
@@ -69,14 +70,32 @@ static const QList<QString> fontFiles = QList<QString>() << QString(":/client/re
                                                          << QString(":/client/resources/fonts/SuisseIntl-Black.otf");
 // static const QString defaultFontFamily("Suisse Int'l");
 
+#ifdef Q_OS_WIN
+static void displayHelpText(const QString &t)  // No console on Windows.
+{
+    static const QString spaces(80, ' ');  // Add a line of non-wrapped space to make the messagebox wide enough.
+    const QString text = QLatin1String("<qt><pre style='white-space:pre-wrap'>") + t.toHtmlEscaped() +
+                         QLatin1String("</pre><pre>") + spaces + QLatin1String("</pre></qt>");
+    QMessageBox::information(0, Theme::instance()->appClientName() + "_HelpText", text);
+}
+
+#else
+
+static void displayHelpText(const QString &t) {
+    std::cout << qUtf8Printable(t);
+}
+#endif
+
+
 AppClient::AppClient(int &argc, char **argv)
     : SharedTools::QtSingleApplication(Theme::instance()->appClientName(), argc, argv),
       _gui(nullptr),
       _theme(Theme::instance()),
       _logExpire(0),
       _logFlush(false),
       _logDebug(false),
-      _debugMode(false) {
+      _debugMode(false),
```

Comment:
Please check the advice of Sonar: https://sonarcloud.io/project/issues?resolved=false&pullRequest=37&id=Infomaniak_desktop-kDrive&open=AY6piGSrMvoBiF_fbDAK.